my-cart/src/app/order.pipe.ts

```diff
@@ -0,0 +1,15 @@
+import { Pipe, PipeTransform } from "@angular/core";
+
+@Pipe({
+    name:'order'
+})
+
+export class OrderPipe implements PipeTransform{
+    transform(inputArray:any[],key:string){
```

Comment:
If this a ordering pipe, `inputArray` could be `unorderedItems` or `items` right? 

Please don't use data types in variable names

Reply:
Changed the input argument